README.md

```diff
@@ -20,42 +20,30 @@
 -->
 
 # Pulsar C++ client library
-<!-- TOC depthFrom:2 depthTo:3 withLinks:1 updateOnSave:1 orderedList:0 -->
 
-- [Requirements](#requirements)
-- [Platforms](#platforms)
-- [Compilation](#compilation)
```

Comment:
Why remove the catalog?

Reply:
Read also https://github.com/apache/pulsar/pull/17470#discussion_r963277120